Dockerfile

```diff
@@ -19,4 +19,4 @@ RUN \
 			/var/tmp/* \
 			/var/lib/apt/lists/*
 WORKDIR /app
-ENTRYPOINT ["python3", "plex_auto_collections.py", "--update"]
\ No newline at end of file
+ENTRYPOINT ["python3", "plex_auto_collections.py"]
```

Comment:
Should we just change to `CMD` instead of `ENTRYPOINT`?

Reply:
I think `ENTRYPOINT` is okay here, since it's not meant to be overridden. The main thing was to just remove `--update`.